include/controls.h

```diff
@@ -0,0 +1,21 @@
+#ifndef CONTROLS_H
+#define CONTROLS_H
+
+#include "SDL.h"
+#include <stdint.h>
+
+// Define the State structure
+typedef struct
```

Comment:
the struct, init_emulator, and emulate8080 are already implemented elsewhere. Just include those files. Unless there a good reason to do this here that I'm missing?